src/org/traccar/Config.java

```diff
@@ -15,48 +15,55 @@
  */
 package org.traccar;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I think we should avoid using this indirect dependency, especially if it's easy to use standard Java.